Eevee.Sleep.Bot/Modules/SlashCommands/RoleManagementSlashModule.cs

```diff
@@ -0,0 +1,69 @@
+using Discord;
+using Discord.Interactions;
+using JetBrains.Annotations;
+using Eevee.Sleep.Bot.Controllers.Mongo;
+using Eevee.Sleep.Bot.Utils;
+
+namespace Eevee.Sleep.Bot.Modules.SlashCommands;
+
+[Group("role", "Commands for managing the roles on a user.")]
+public class RoleManagementSlashModule : InteractionModuleBase<SocketInteractionContext> {
+    [SlashCommand("change", "Select a specific one from the role you have")]
+    [UsedImplicitly]
+    public Task ChangeRoleAsync() {
+        return Context.Interaction.RespondAsync(
+            components: DiscordMessageMaker.MakeRoleSelectButton(
+                hasRoles: DiscordRoleContoller.FindRoleIdsByUserId(Context.User.Id), 
+                guildRoles: Context.Guild.Roles),
```

Comment:
Parenthesis line break - The `)` after `Context.Guild.Roles` should be in the next line.